library/src/main/java/ru/terrakok/cicerone/CommandBuffer.java

```diff
@@ -15,14 +18,15 @@
  */
 class CommandBuffer implements NavigatorHolder {
     private Navigator navigator;
-    private Queue<Command[]> pendingCommands = new LinkedList<>();
+    private final Queue<Command[]> pendingCommands = new LinkedList<>();
+    private final Handler mainHandler = new Handler(Looper.getMainLooper());
 
     @Override
     public void setNavigator(Navigator navigator) {
         this.navigator = navigator;
         while (!pendingCommands.isEmpty()) {
             if (navigator != null) {
-                executeCommands(pendingCommands.poll());
+                navigator.applyCommands(pendingCommands.poll());
```

Comment:
Am I correct understand the code is taken from the latest version of the library?

Reply:
Taking into account that new code is in Kotlin while this is Java: https://github.com/terrakok/Cicerone/blob/master/library/src/main/kotlin/com/github/terrakok/cicerone/CommandBuffer.kt